### PR TITLE
add spacy-models 3.3.0

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Notes:
- fixes https://github.com/conda-forge/spacy-model-en_core_web-feedstock/issues/13
- probably needs some better tests
  - maybe re-generate, re-render and compare recipe during test
- would want feedback from other maintainers on other best practices for keeping these up-to-date

cc @wolfv @h-vetinari

### CODE REVIEW

File: recipes/spacy-models/meta.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+package:
+  name: spacy-models
+  version: {{ version }}
+
+source:
+  - url: https://raw.githubusercontent.com/explosion/spaCy/master/LICENSE
+    sha256: c07800e058b1b544eb9d47dd81687582fe6830c1ddc47f57521fc93628685915
+<% for p, m in all_metas.items() %>
+  - url: https://github.com/explosion/spacy-models/releases/download/<<m.lang >>_<< m.name >>-{{ version }}/<< m.lang >>_<< m.name >>-{{ version }}-tar.gz
```
Comment (bollwyvl) [inline recipes/spacy-models/meta.yaml.j2:14]: should be `.tar.gz`, not `-tar.gz`

**Comment by bollwyvl** [comment]
Hm... not getting lots of logs back... might need to:
- start with a smaller subset of languages/sizes
- wait to split up into multiple builds in feedstock

perhaps demonstrating the  approach works with just `*_sm` would be sufficient.

**Comment by bollwyvl** [comment]
Nice:
```
- Encountered problems while solving:
-   - nothing provides requested sudachidict-core >=20211220
-   - nothing provides requested sudachipy >=0.5.2,!=0.6.1
-   - nothing provides sudachidict-core >=20211220 needed by spacy-model-ja_core_news_sm-3.3.0-pyhd8ed1ab_0

```

**Comment by bollwyvl** [comment]
fellow maintainers, what do you think about this?

@conda-forge/spacy-model-de
@conda-forge/spacy-model-en_vectors_web
@conda-forge/spacy-model-es
@conda-forge/spacy-model-fr
@conda-forge/spacy-model-pt
@conda-forge/spacy-model-xx
@conda-forge/spacy-model-nl
@conda-forge/spacy-model-nl

Barring any serious arguments:
@conda-forge/staged-recipes ready for review

File: recipes/spacy-models/meta.yaml.j2 [hunk 2]
```diff
@@ -0,0 +1,80 @@
+{% set version = "<< version >>" %}
```
Comment (ocefpaf) [inline recipes/spacy-models/meta.yaml.j2:1]: Clever! A recipe to generate recipes ;-p

**Comment by FernandezMathieu** [comment]
Well done, this approach is more clever. What's the plan for versions under 3.9 ?

**Comment by bollwyvl** [comment]
> What's the plan for python versions under 3.9 ?

These are all noarch with a bottom ping of `3.6`... or will be once we figure out https://github.com/conda-forge/spacy-models-feedstock/issues/1... if there are more issues (or you'd like to help maintain) chime in over on a new issue over there!

**Comment by bollwyvl** [comment]
An update here: still fighting with some plumbing on https://github.com/conda-forge/feedstock-outputs/pull/29 but getting close to new builds!

**Pull request merged**